lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/domain/KBVItem.java

```diff
@@ -3,12 +3,14 @@
 import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
 import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
 
+import java.util.UUID;
+
 @DynamoDbBean
 public class KBVItem {
-    private String sessionId;
+    private UUID sessionId;
     private String token;
     private String questionState;
-    private String expiryDate;
+    private Long expiryDate;
```

Comment:
As we are intending on setting the expiryDate to a null value then use `long` (value type) rather than the reference type

Reply:
Done!